src/en/allanime/src/eu/kanade/tachiyomi/extension/en/allanime/Dto.kt

```diff
@@ -53,7 +53,7 @@ class SearchManga(
 ) {
     fun toSManga() = SManga.create().apply {
         title = englishName ?: name
-        url = "/manga/$id/${name.titleToSlug()}"
```

Comment:
this would break the "in library mark", override `getMangaUrl` and `getChapterUrl` instead to return correct webview/share urls